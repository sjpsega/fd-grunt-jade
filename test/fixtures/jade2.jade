!!!5
html
    body
        #header.center 你好