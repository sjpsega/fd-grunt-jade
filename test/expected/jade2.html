<!-- create by jade! -->
<!DOCTYPE html>
<html>
  <body>
    <div id="header" class="center">你好</div>
  </body>
</html>